test/FastTests/Issues/RavenDB_19266.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+using Raven.Client.ServerWide.Operations.DocumentsCompression;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FastTests.Issues
+{
+    public class RavenDB_19266 : RavenTestBase
+    {
+        public RavenDB_19266(ITestOutputHelper output) : base(output) { }
+
+        public static string RandomString(Random random, int length)
+        {
+            const string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+            return new string(Enumerable.Repeat(chars, length)
+                .Select(s => s[random.Next(s.Length)]).ToArray());
+        }
+
+        internal class Order
+        {
+            public List<string> Lines = new();
+        }
+
+        [Theory]
```

Comment:
It needs to be decorated with `[LicenseRequiredTheory]` or moved to SlowTests (preferably). FastTests are run by github actions where we use AGPL license so compression is not enabled and we got failure on this test:

https://github.com/ravendb/ravendb/runs/8092295608?check_suite_focus=true

Reply:
Done.